vanilla_installer/defaults/users.py

```diff
@@ -95,10 +95,10 @@ def __on_username_entry_changed(self, *args):
             _status = False
             self.__window.toast("Username cannot be empty. Please type a username.")
 
-        # cannot be root
-        elif _input == "root":
+        # cannot be root, debian, vanillaos or vanilla-os
+        elif _input == "root" or _input == "vanillaos" or _input == "vanilla-os" or _input == 'debian':
```

Comment:
@RudraSwat a small suggestion regards to reserved usernames. Can you put all the reserved usernames into a list and update the conditional to check in the list. This will prevent lengthy conditional statement. Something like below:

```python
reserved_usernames = ['root', ...]
elif _input in reserved_usernames:
    ....
```

or if you can get list of reserved usernames from a util function will be even better.